builtin/logical/pki/path_tidy.go

```diff
@@ -164,6 +194,11 @@ func (b *backend) startTidyOperation(req *logical.Request, config *tidyConfig) {
 				}
 			}
 
+			// Check for cancel before continuing.
```

Comment:
If anything, I think we just need a `atomic.CompareAndSwapUint32(b.tidyCancelCAS, 1, 0)` at the top of this function before _getting_ the `tidyCASGuard`. otherwise, I don't think it really matters, IMO?

Reply:
Sure that would work as well.

Reply:
(I think that's correct because it ensures that if a cancel somehow came in _after_ we finished, we'd want any new ones to start fresh).